File: rb/fizzbuzz.rb
def fizzbuzz(num)
  (1..num).each do |n|
    if n % 15 == 0
      puts "fizzbuzz"
    elsif n % 5 == 0
      puts "buzz"
    elsif n % 3 == 0
      puts "fizz"
    else
      puts n
    end
  end
end

fizzbuzz(100)

File: js/fizzBuzz.js
var fizzbuzz = function(num) {
  for (var i = 0; i <= num; i ++) {
    if (i % 15 == 0) {
      console.log("fizzbuz");
    }
    else if (i % 5 === 0) {
      console.log("buzz");
    }
    else if (i % 3 === 0) {
      console.log("fizz");
    }
    else {
      console.log(i);
    }
  };
}

fizzbuzz(100)
